src/index.ts

```diff
@@ -45,12 +59,12 @@ const main = async () => {
   const zeekFlag = Boolean(process.argv.filter((arg) => arg === "--zeek")[0]);
   const l1RpcUrl = String(
     process.argv
-      .filter((arg) => arg.startsWith("l1-rpc-url"))
+      .filter((arg) => arg.startsWith("--l1-rpc-url"))
```

Comment:
Given we are using the full argument name, why not comparing the strings instead of .startsWith?

Reply:
AH, now I know why it doesn't work for me... we are expecting people to pass arguments with an "=" instead of just space. Is this a good way to do it?

Reply:
But shouldn't we change the approach to the expected behaviour of CLI commands?